simulation/Map.py

```diff
@@ -0,0 +1,17 @@
+from simulation.utils import Vehicle, Crossroad
+from typing import List
+
+class Map:
+    def __init__(self):
+        self.vehicles:List[Vehicle] = []
+        self.crossroads:List[Crossroad] = []
```

Comment:
Sporym krokiem będzie stworzenie pełnej kompozycji klas. Trochę opisałem w #8 jak można to zrobić - połączyć odpowiednią parę skrzyżowań dwoma drogami jednokierunkowymi, a na każdej z nich stworzyć kolejki pojazdów + sygnalizację świetlną na końcu. 

Klasa Simulation powinna zajmować się zmianą stanu sygnalizacji, aktualizowaniem pozycji pojazdów oraz przemieszczaniem samochodów z jednej kolejki (jak dotrą na jej koniec i światło jest zielone) na koniec drugiej (wybranej lub wylosowanej).